src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -1492,9 +1492,36 @@
   <div {{ block('widget_attributes') }}>
     {% for group, buttons in button_groups %}
       <div class="btn-group group-{{ group }}">
-        {% for button in buttons %}
-          {{- form_widget(attribute(form, button)) -}}
+        {# Render inline actions #}
+        {% set maxInlineButtons = form.vars.max_inline_buttons %}
+        {% for action in form.children %}
+          {% if loop.index <= maxInlineButtons %}
```

Comment:
As commented in the form type the default behaviour should be no dropdown when `maxInlineButtons` is `false`

```suggestion
          {% if maxInlineButtons is same as(false) or loop.index <= maxInlineButtons %}
```